types.d.ts

```diff
@@ -30,6 +30,7 @@ export interface BlogSettings {
   lang?: string;
   timezone?: string;
   canonicalUrl?: string;
+  unocss?: any // "object"
```

Comment:
nice, can you please import the `UserConfig` types?

Reply:
Sorry, not sure what you mean

Reply:
i mean you can instead the `unocss?: any ` with `unocss?:UserConfig`, and the `UserConfig` can be imported by `import type { UserConfig as UnoConfig } from "https://esm.sh/@unocss/core@0.43.2";`